README.md

```diff
@@ -72,102 +72,70 @@ For guidance on developing for PowerToys, please read the [developer docs](/doc/
 
 Our [prioritized roadmap][roadmap] of features and utilities that the core team is focusing on.
 
-### 0.66 - December 2022 Update
+### 0.67 - January 2023 Update
 
-In this release, we focused on stability and improvements.
+In this release, we focused on releasing new features and improvements.
 
 **Highlights**
 
-- PowerToy utilities now ship with self-contained .NET 7, meaning it's not necessary to install .NET as part of the installer and it's easier to keep up to date.
-- It's possible to pick which of the installed OCR languages is used by Text Extractor by selecting it in the right-click context menu.
-- Added a setting to sort the order of the accented characters by usage frequency in Quick Accent.
+- Added an option for PowerToys Run to tab through results instead of context buttons.
+- All PowerToys registry entries are moved from HKLM to HKCU.
+- New utility: Quick access system tray launcher
 
-### General
+### Awake
 
-- Reduced resource consumption caused by logging. A thread for each logger was being created even for disabled utilities.
-- The .NET 7 dependency is now shipped self-contained within the utilities, using deep links to reduce storage space usage.
-
-### Color Picker
-
-- Fixed an issue where the custom color formats were not working when picking colors without using the editor.
-- Fixed a crash when using duplicated names for color formats.
-- Added two decimal formats, to distinguish between RGB and BGR.
-- Fixed color name localization, which was not working correctly on 0.65.
+- Disable instead of hiding "Keep screen on" option.
 
 ### FancyZones
 
-- Fixed an editor crash caused by deleting a zone while trying to move it.
-- Reduce the time it takes the tooltip for layout shortcut setting to appear in the editor.
+- Refactored and improved code quality.
 
-### File Locksmith
+### File explorer add-ons
 
-- Fixed an issue causing File Locksmith to hang when looking for open handles in some machines.
+- Fixed escaping HTML-sensitive characters when previewing developer files. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
 
-### Hosts File Editor
+### Image Resizer
 
-- Added a warning when duplicated entries are detected. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
+- Fixed call to AssocGetPerceivedType.
```

Comment:
Here the context is better to call out as "Improved code quality around a silent crash that was being reported to Microsoft servers."